modules/nw-ipfailover-configuration.adoc

```diff
@@ -16,20 +16,20 @@ For production use, set a `selector` that selects at least two nodes, and set `r
 
 .Prerequisites
 
-* You are logged in to the cluster with a user with `cluster-admin` privileges.
+* You are logged in to the cluster as a user with `cluster-admin` privileges.
 * You created a pull secret.
+* {rh-openstack-first} only: You installed the link:https://docs.openstack.org/python-openstackclient/latest/[{rh-openstack} client ({op-system} documentation)] on the target environment.
```

Comment:
better 'You need to install a'

Reply:
These prereqs are typically worded in the past tense and they are mandatory.